src/utils.ts

```diff
@@ -209,6 +209,13 @@ export function error(...messages: any[]): void {
   console.error(getStyledTextForLogging('red')?.('error'), ...messages)
 }
 
+export function warning(...messages: any[]): void {
+  if (!process.env.VERBOSE) {
+    return console.warn(messages)
+  }
+  console.warn(getStyledTextForLogging('yellow')?.('warning'), ...messages)
```

Comment:
This will always log. If you look at the `log` function below it actually returns `void` when its NOT verbose.

Since this is just a warning I recommend the following change:
```suggestion
  if (!process.env.VERBOSE) {
    return
  }
  console.warn(getStyledTextForLogging('yellow')?.('warning'), ...messages)
```

Reply:
This was written to mimic https://github.com/bcherny/json-schema-to-typescript/blob/6adcad98ac28334f1e8cd932e412c64dde4205f5/src/utils.ts#L205-L210 

I don't pretend to understand that choice and it's not my aim to question it. I'm just following precedence here.